homeassistant/components/tuya/number.py

```diff
@@ -252,6 +252,17 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Thermostat
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf45ld5l0t9
+    "wk": (
+        NumberEntityDescription(
+            key=DPCode.ROOM_TEMPERATURE_CALIBRATION,
```

Comment:
I cannot find this data point in the linked API specification above. Am I missing something?

Reply:
As above.

Reply:
As above.